src/BucketTesting/Validation/CampaignUtilizationValidator.php

```diff
@@ -105,6 +105,12 @@ private function findInconsistentFeatureToggleOccurrences(): void {
 	}
 
 	private function getCampaignNameFromFeatureToggle( string $featureToggle ): string {
-		return substr( $featureToggle, 0, strrpos( $featureToggle, '.' ) );
+		$strpos = strrpos( $featureToggle, '.' );
+		if ( $strpos === false ) {
```

Comment:
@gbirke Should I re-throw the Exception as we are doing in the case of JsonException?

Reply:
`strrpos` does not throw an exception like `json_encode`: https://www.php.net/manual/en/function.strrpos
Therefore *have* to check for false instead of catching and re-throwing.